jmh-tests/src/main/java/org/eclipse/collections/impl/jmh/LongLongMapLargeStressTest.java

```diff
@@ -246,7 +246,7 @@ public void kolobokeRemove()
             {
                 newMap.remove(this.kolobokeLongKeysForMap[i]);
             }
-            if (newMap.size() != 0)
+            if (!newMap.isEmpty())
```

Comment:
`notEmpty()`?

Reply:
The class is coming from net.openhft.koloboke.collect.map.hash package and does not offer a notEmpty() method.